### PR TITLE
Feature: Enabled support for inputting custom icon path

### LINKED ISSUE #15941: Can't type in text field in customization in a folder properties
### Description

I was going to change the icon of a folder but it wouldn't let type inn the text field it would only let me select icons from shell32.dll.
There is a workaround tho, by clicking the "Browse" button i can select my icon file but faster for me to just type it out :(

### Steps To Reproduce

1. Right click a folder and select properties.
2. Go to customization menu.
3. Try to type into the text field and it wouldn't work :(

### Requirements

It wouldn't let me type in the text field in the customization menu in a folder properties

### Files Version

3.5.0.0

### Windows Version

22631.3958

### Log File

[debug.log](https://github.com/user-attachments/files/16489658/debug.log)

### PR DESCRIPTION
**Pull request by yaira2**
**Resolved / Related Issues**

To prevent extra work, all changes to the Files codebase must link to an approved issue marked as `Ready to build`. Please insert the issue number following the hashtag with the issue number that this Pull Request resolves.
- Closes #15941

**Steps used to test these changes**

Stability is a top priority for Files and all changes are required to go through testing before being merged into the repo. Please include a list of steps that you used to test this PR.

1. Confirmed entering path loads dll file in properties window

### CODE REVIEW

File: src/Files.App/ViewModels/Properties/CustomizationViewModel.cs [hunk 1]
```diff
@@ -58,16 +71,14 @@ public CustomizationViewModel(IShellPage appInstance, BaseProperties basePropert
 			else
 				return;
 
+			DllIcons = [];
```
Comment (0x5bfa) [inline src/Files.App/ViewModels/Properties/CustomizationViewModel.cs:74]: Can you move this to L24?

**Review by 0x5bfa — APPROVED** [review]
Otherwise, looks good

File: src/Files.App/ViewModels/Properties/CustomizationViewModel.cs (hunk 1 above)
Comment (yaira2) [reply src/Files.App/ViewModels/Properties/CustomizationViewModel.cs:74]: Like this?
`public ObservableCollection<IconFileInfo> DllIcons { get; } = [];`

**Review by 0x5bfa — APPROVED** [review]
Looks good to me, code wise.

**Pull request merged**